foundry.toml

```diff
@@ -4,9 +4,12 @@ test = "test/forge"
 libs = ["lib"]
 evm_version = "paris"
 via-ir = true
+optimizer = true
+optimizer_runs = 200
+bytecode_hash = "none"
 fs_permissions = [
-    { access = "read", path = "./config/"},
-    { access = "read", path = "./out/"}
+    { access = "read", path = "./config/" },
```

Comment:
```suggestion
```

Reply:
Do you remember why either read access was granted ? I can't seem to find what breaks if we remove `fs_permission` altogether

Reply:
Removing optimistically, as it's not hard to reintroduce it